blockchain/store.go

```diff
@@ -63,7 +63,7 @@ func (bs *BlockStore) LoadBlock(height int64) *types.Block {
 		part := bs.LoadBlockPart(height, i)
 		buf = append(buf, part.Bytes...)
 	}
-	err := cdc.UnmarshalBinary(buf, block)
+	err := cdc.UnmarshalBinaryLengthPrefixedBinary(buf, block)
```

Comment:
Oh no, this should actually be named: `UnmarshalBinaryLengthPrefixed` instead 🙈 Not sure how me and two reviewers didn't see that. I'll delete the release 0.13.0 for now and add commit to master that fixes that (and tag this again).

Reply:
https://github.com/tendermint/go-amino/pull/236